### ISSUE TITLE
Tipping Banner needs better keyboard accessibility - Follow up to 6147

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#6986]
## Description 
Found while testing #6147 

When you bring up the tip banner (and have enough BAT) the `Send Tip` button is in focus as indicated by the outline on the button. However, if you attempt to tab to choose another amount other than the pre-selected one, you exit the tip banner and there is no way to tab back into the tip banner. 

## Steps to Reproduce

   1. Go to a site and open the tip banner.
   2. If you have enough BAT for the pre-selected amount, the `Send Tip` button is in focus. If not, tab once so that the `add funds` link is in focus.
   3. Tab again.
   4. URL bar is now in focus. No matter how many times you select tab, you will never get back into the tip banner. 

## Actual result:
Selecting tab from the last tab-selectable item on the tip banner (either `Send Tip` or `add funds`) causes you to exit the tip banner and go to URL bar. Now, you are unable to tab back into the tip banner. You must use a mouse to get back into the tip banner.

## Expected result:
I would expect tab to take me to the first tab selectable item in the banner, I believe this is the X in the circle in the upper right corner. 

## Reproduces how often: 
easily

## Brave version (brave://version info)

Brave | 1.1.4 Chromium: 78.0.3904.97 (Official Build) beta (64-bit)
-- | --
Revision | 021b9028c246d820be17a10e5b393ee90f41375e-refs/branch-heads/3904@{#859}
OS | macOS Version 10.13.6 (Build 17G5019)

## Version/Channel Information:

- Can you reproduce this issue with the current release? n/a
- Can you reproduce this issue with the beta channel? yes
- Can you reproduce this issue with the dev channel? yes
- Can you reproduce this issue with the nightly channel? yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:
cc @brave/legacy_qa 
cc @mandar-brave @alexwykoff as this relates to tipping efficiency changes

**Comment by LaurenWags** [comment]
Still occurs as of the below version:

```

Brave | 1.21.74 Chromium: 89.0.4389.72 (Official Build) (x86_64)
-- | --
Revision | 3f345f156bfd157bd1bea06310e55f3fb2490359-refs/branch-heads/4389@{#1393}
OS | macOS Version 10.15.7 (Build 19H524)

```

**Comment by Miyayes** [comment]
Note: Tipping banner has probably changed since this issue was originally filed. So, check if this still applies.

**Comment by emerick** [comment]
Unable to repro. I think the most recent redesign of the tipping banner ended up addressing this, but please reopen if I'm incorrect.
